src/fal/cli/flow_runner.py

```diff
@@ -225,47 +231,41 @@ def _unique_ids_to_model_names(id_list: List[str]) -> List[str]:
     return list(map(lambda id: _unique_id_to_model_name(id), id_list))
 
 
-def _combine_fal_run_results(target_path):
-    result_files = _get_fal_run_results(target_path)
+def _combine_fal_run_results(target_path: str) -> None:
+    target_path = Path(target_path)
+    dbt_run_results, fal_run_results = [], []
+    for path in target_path.glob("fal_results_*.json"):
+        assert path.is_file()
 
-    if not result_files:
-        return
+        results = _get_all_result_content(path)
 
-    all_results = reduce(
-        lambda all, next: all.extend(_get_result_array(next)) or all,
-        result_files,
-        [],
-    )
+        if results.get(SCHEMA_KEY) != "fal":
+            dbt_run_results.append(results)
+        fal_run_results.append(results)
```

Comment:
Shouldn't there be an else here?

Reply:
I think it opens up the question of what 'fal_results.json' is. Does it only contain Fal's own stuff (like post-hooks and Python model script executions) or does it contain both DBT run results and fal's stuff (like a fully fledged thing). WDYT?

CC: @turbo1912 @chamini2